roles.json

```
@@ -267,6 +267,25 @@
             ]
         }
     },
+    {
+        "role": "DevOps and Operations",
+        "url": "devops_team",
+        "people": [
+            "Madison Bray",
+            "Pey Lian Lim",
+            "Brigitta Sip\u0151cz"
+        ],
+        "role-head": "DevOps and Operations",
+        "responsibilities": {
+            "description": "Ensure the smooth running of the project",
+            "details": [
+                "Set up and maintain continuous integration services",
+                "Ensure adequate labelling of issues and pull requests",
```

If we are doing US English, it's only one "l". ⚔️ 
```suggestion
                "Ensure adequate labeling of issues and pull requests",
```